src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -94,6 +96,21 @@ public void storeMessage(Message message) {
     return processQuery(query);
   }
 
+  /**
+   * Gets reviews posted for a specific city.
+   *
+   * @return a list of messages posted for a city, or empty list if there is no review for that city.
+   * List is sorted by time descending.
+   */
+  public List<Message> getMessagesForCity(String city) {
```

Comment:
Nit: instead of calling it messages, can consider renaming it to reviews. So for example, getMessagesForCity -> getReviewsForCity. The underlying class can still be called Message, since it is shared.

Reply:
Done